sdk/src/types/api/core.rs

```diff
@@ -402,6 +402,28 @@ pub enum BlockFailureReason {
     Invalid = 255,
 }
 
+impl core::fmt::Display for BlockFailureReason {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            Self::TooOldToIssue => write!(f, "The block is too old to issue."),
```

Comment:
I wonder if there is a way to make these as constants available to the bindings to avoid the duplication, if ever that's worth it...

Reply:
Unfortunately I don't think there's a particularly good solution to that. :/

Reply:
Thought so, never mind